python/lsst/display/firefly/firefly.py

```diff
@@ -125,6 +125,13 @@ def __init__(self, display, verbose=False, url=None,
     def _getRegionLayerId(self):
         return "lsstRegions%s" % self.display.frame if self.display else "None"
 
+    def _clear_image(self):
+        """Delete the current image in the Firefly viewer
+        """
+        self._client.dispatch_remote_action(channel=self._client.channel,
+                                            action_type='ImagePlotCntlr.deletePlotView',
+                                            payload=dict(plotId=str(self.display.frame)))
```

Comment:
I understand that you trying calling this in `_mtv` and it didn't help with this particular problem, but if it does no harm it may be exactly what's needed to fix [DM-14695](https://jira.lsstcorp.org/browse/DM-14695), in that it seems to be a programmatic way to do exactly what I was doing in the GUI in my workaround (though I gather there may be different fix for that issue already in the works).

Reply:
It does fix the mask but it deletes the image entirely. @robyww and I looked at this and decided this was undesirable. Once you have, say, three displays up, and you `mtv` again to the first one, displays 2 and 3 shift to the left and display 1 appears on the right. Trey has a server-side fix that we will do instead.